test/unit/pusher_test.rb

```diff
@@ -109,7 +109,7 @@ class PusherTest < ActiveSupport::TestCase
       @gem = gem_file("bad-date-1.0.0.gem")
       @cutter = Pusher.new(@user, @gem)
       @cutter.process
-      assert_match(/There was a problem saving your gem. year too big to marshal: 1017 UTC/, @cutter.message)
+      assert_match(/There was a problem saving your gem. year too (big|small) to marshal: 1017 UTC/, @cutter.message)
```

Comment:
:thinking: What about to write a little todo to remove this once Ruby 2.6 becomes default one?

Reply:
Done!